src/ServicePulse.Host/vue/src/resources/UpdateEmailNotificationsSettingsRequest.ts

```diff
@@ -1,8 +1,8 @@
 export default interface UpdateEmailNotificationsSettingsRequest {
   smtp_server: string;
   smtp_port: number;
-  authentication_account: string;
-  authentication_password: string;
+  authorization_account: string;
+  authorization_password: string;
```

Comment:
These are Authentication but I see they match what ServiceControl expects. https://github.com/Particular/ServiceControl/blob/master/src/ServiceControl/Notifications/Api/UpdateEmailNotificationsSettingsRequest.cs

Reply:
yes this is confusing, we should actually fix ServiceControl